src/main/java/org/kohsuke/github/GHRepositorySearchBuilder.java

```diff
@@ -76,8 +76,8 @@ public GHRepositorySearchBuilder forks(String v) {
      *      in forks</a>
      *
      */
-    public GHRepositorySearchBuilder fork(Fork fork) {
-        if (Fork.PARENT_ONLY.equals(fork)) {
+    public GHRepositorySearchBuilder fork(GHFork fork) {
```

Comment:
This is reasonable but to maintain backward compatibility we need to keep `Fork` and the related method and mark them both `@Deprecated`.

Reply:
Thanks a lot for your feedback. I have updated PR with the requested changes. Please kindly have another look and kick-off the build checks when you have some time :)